### PR TITLE
Stringtable Zeus and View Distance

### PR DESCRIPTION
**Pull request by gienkov**
* Stringtabled ACE Zeus module
* Stringtabled ACE View Distance
* Lowered default view distance to 1500 because default value (10000)
makes enormous FPS drop at start for low end machines

### CODE REVIEW

File: addons/viewdistance/stringtable.xml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Zeus">
+        <Container ID="ZeusModule">
+            <Key ID="STR_ACE_ViewDistance_Module_DisplayName">
+                <English>View Distance Limiter</English>
+                <Polish>Ogranicznik zasięgu widzenia</Polish>
+            </Key>
+			<Key ID="STR_ACE_ViewDistance_Module_Description">
```
Comment (esteldunedain) [inline addons/viewdistance/stringtable.xml]: tabs

File: addons/viewdistance/stringtable.xml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Zeus">
+        <Container ID="ZeusModule">
```
Comment (esteldunedain) [inline addons/viewdistance/stringtable.xml]: Package and container entries are wrong. In fact most ACE module don't use the container entry, as it has no use

**Comment by esteldunedain** [comment]
> Lowered default view distance to 1500 because default value (10000) makes enormous FPS drop at start for low end machines

No, that's plain wrong! ACE_ViewDistance doens't override the view distance you have set on your graphical settings. It's totally independent from that. 

Please revert that

Quick example:
````Suppose Player A has his viewdistance to 5000m and Player B has his to 2000m:
- If ACE_ViewDistance is set to 10000m (default), A can see 5000m and B 2000m.
- If ACE_ViewDistance is set to 1500m (default), A can see 1500m and B 1500m.
```

So leaving the default to 10000m is like saying "Everyone can see whatever they configure on their graphical settings". That should be the default

File: addons/viewdistance/ACE_Settings.hpp [hunk 3]
```diff
@@ -2,47 +2,45 @@ class ACE_Settings {
...
-        displayName = "Enable ACE viewdistance";
-        description = "Enables ACE viewdistance module.";
+        displayName = "$STR_ACE_ViewDistance_enabled_DisplayName";
+        description = "$STR_ACE_ViewDistance_enabled_Description";
     };
     class GVAR(viewDistanceOnFoot) {
         typeName = "SCALAR";
         isClientSettable = 1;
-        value = 11; // index, NOT value // Can set it to client's actual viewdistance in the init function once ACE_Settings supports numbers (if ever).
+        value = 0; // index, NOT value // Can set it to client's actual viewdistance in the init function once ACE_Settings supports numbers (if ever).
```
Comment (esteldunedain) [inline addons/viewdistance/ACE_Settings.hpp]: Please revert this change as per comment below.

File: addons/viewdistance/ACE_Settings.hpp [hunk 4]
```diff
@@ -2,47 +2,45 @@ class ACE_Settings {
...
-        displayName = "Client View Distance (On Foot)";
-        description = "Changes in game view distance when the player is on foot.";
+        displayName = "$STR_ACE_ViewDistance_onFoot_DisplayName";
+        description = "$STR_ACE_ViewDistance_onFoot_Description";
     };
     class GVAR(viewDistanceLandVehicle) {
         typeName = "SCALAR";
         isClientSettable = 1;
-        value = 11; // index, NOT value
+        value = 0; // index, NOT value
```
Comment (esteldunedain) [inline addons/viewdistance/ACE_Settings.hpp]: Idem

File: addons/viewdistance/ACE_Settings.hpp [hunk 5]
```diff
@@ -2,47 +2,45 @@ class ACE_Settings {
...
-        displayName = "Client View Distance (Land Vehicle)";
-        description = "Changes in game view distance when the player is in a land vehicle.";
+        displayName = "$STR_ACE_ViewDistance_landVehicle_DisplayName";
+        description = "$STR_ACE_ViewDistance_landVehicle_Description";
     };
     class GVAR(viewDistanceAirVehicle) {
         typeName = "SCALAR";
         isClientSettable = 1;
-        value = 11; // index, NOT value
+        value = 0; // index, NOT value
```
Comment (esteldunedain) [inline addons/viewdistance/ACE_Settings.hpp]: Idem

File: addons/zeus/stringtable.xml [hunk 6]
```diff
@@ -0,0 +1,71 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Zeus">
+        <Container ID="ZeusModule">
+            <Key ID="STR_ACE_Zeus_Module_DisplayName">
+                <English>Zeus Settings [ACE]</English>
+                <Polish>Ustawienia Zeusa [ACE]</Polish>
+            </Key>
+			<Key ID="STR_ACE_Zeus_Module_Description">
```
Comment (esteldunedain) [inline addons/zeus/stringtable.xml]: tabs

**Comment by gienkov** [comment]
Fixed all.

**Comment by gienkov** [comment]
@esteldunedain please explain to me why then setting this to for example 10000 (which **sets by itself by default**) makes my draw distance increase to 10000 even when inside my graphics setting I have set 1500? It **defaults** draw distance to 10000 and makes game cumbersome because of that. Default value that is set **should be lower**.

**Comment by esteldunedain** [comment]
> @esteldunedain please explain to me why then setting this to for example 10000 

Mmm, yes, it's not working as intended. We'll have to default to an "unforced" view distance.

**Comment by SilentSpike** [comment]
Zeus was done here: #1295

Should probably remove it on this side as the file was otherwise changed quite a bit as part of that PR and it'll mean no merge conflicts between these two PR

**Comment by SilentSpike** [comment]
Though, since you have more translations perhaps not

**Comment by SilentSpike** [comment]
@esteldunedain I believe arma doesn't use the player's graphical view distance setting when it comes to MP (that's why a bunch of view distance addons popped up originally).

**Comment by esteldunedain** [comment]
@gienkov, should be fixed in #1311

**Comment by gienkov** [comment]
* Merge conflicts fixed.
* Added translation for "Video Settings".

**Comment by gienkov** [comment]
Can dis go in already?
